github-actions/summarize-errors/src/types.d.ts

```diff
@@ -0,0 +1,38 @@
+export interface HimalayaNode {
```

Comment:
`himalaya` doesn't have a typing module for typescript, but it lists its schema at this link: https://github.com/andrejewski/himalaya/blob/master/text/ast-spec-v1.md, so I recreated it here.